src/Components/LoginPage/LoginPageComponents/Forms/SignInForm.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import {Formik, Form} from 'formik';
+import * as Yup from 'yup';
+import {SignFormInput} from "../SignFormInput";
+import {Link} from "react-router-dom";
+import {RESTORE_PASSWORD_PATH, SIGN_IN_PATH} from "../../path";
+import { useHistory } from "react-router-dom";
+
+
+
+export const SignInForm = (props) => {
+    const signBarTitle = `Sign In`
+    const signBarButtonText = 'Sign In'
+
+    const history = useHistory();
+
+    function handleClick() {
+        history.push("/");
+    }
+
+    return (
+        <>
+            <Formik
+                initialValues={{
+                    email: '',
+                    password: '',
+                }}
+                validationSchema={Yup.object({
+                    email: Yup.string()
```

Comment:
Please move validation scheme to validation.js in this folder and export from there

Reply:
done